src/card/generate_card_assets.rb

```diff
@@ -1,9 +1,27 @@
 require_relative "./models.rb"
 require_relative "./generators.rb"
 require_relative "./catalogue_adapters.rb"
+require_relative "./storage_adapters.rb"
+require "tmpdir"
 
 catalogue_adapter = Card::CatalogueAdapters::JsonCatalogueAdapter.new("data/card_catalogue.json")
-
 generator = Card::Generators::VipsBasic.new
+storage_adapter = Card::StorageAdapters::GitStorageAdapter.new
+
+card_specs = catalogue_adapter.printable_cards
+
+Dir.mktmpdir do |tmp_dir|
```

Comment:
I'm using a tmp directory here since the generator needs a place to write the printed cards. It doesn't do much with this configuration since afterwards we just copy it into the assets folder, but with other storage adapters then it'd be useful

Reply:
looks good - I'd hazard a guess that Google Drive would be the avg user's hosting/working dir. Perhaps down the line we could take another look at generating public direct links from Drive files